gulp-core-build-typescript/src/TSLintTask.ts

```diff
@@ -115,6 +117,10 @@ export class TSLintTask extends GulpTask<ITSLintTaskConfig> {
     const taskScope: TSLintTask = this;
 
     const activeLintRules: any = taskScope._loadLintRules(); // tslint:disable-line:no-any
+
+    // @todo This stream is not piped to returned value. Find a way to block the execute flow.
+    this._writeLintRules(activeLintRules);
```

Comment:
I can do it as *gulp-like* pattern like [this](http://stackoverflow.com/a/23398200/1436671). Do you think it is OK?